pkg/cri/resource-manager/cache/container.go

```diff
@@ -25,6 +25,11 @@ import (
 	cri "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
+const (
+	resmgrLabelNs      = "cri-resource-manager.intel.com/"
+	resmgrAnnotationNs = "cri-resource-manager.intel.com/"
```

Comment:
I think these should be upper in our internal pkg-hierarchy. Although directly at cri/resource-manager would probably be the best, since we put the default socket paths to cri/resource-manager/sockets, maybe we should put these to cri/resource-manager/kubernetes. Something like pkg/cri/resource-manager/kubernetes/metadata.go: (sans my typos)
```

import (
...
    v1 "k8s.io/api/core/v1"
...
)

const (
...
    KeyNamespace = "cri-resource-manager.intel.com"
...
)

func ResourceManagerKey(name string) string {
    return KeyNamespace + "/" + name
}

func InResourceManagerNamespace(key string) bool {
    return strings.HasPrefix(key, KeyNamespace + "/")
}

func IsResourceManagerLabel(key string) bool {
    return InResourceManagerNamespace(key)
}

func IsResourceManagerAnnotation(key string) bool {
    return InResourceManagerNamespace(key)
}

func IsResourceManagerResource(name v1.ResourceName) bool {
    return InResourceManagerNamespace(string(name))
}

...
```

Reply:
> `com.intel.cri-resource-manager/` ?
> not sure nowadays, which will be more readable/best practice.

I guess it's a matter of taste/personal preference. In my experience what I used is more common in k8s, and, I personally find it more readable nowadays

Reply:
> I think these should be upper in our internal pkg-hierarchy. Although directly at cri/resource-manager would probably be the best, since we put the default socket paths to cri/resource-manager/sockets, maybe we should put these to cri/resource-manager/kubernetes. Something like this: (sans my typos)
> 
> ```
> // pkg/cri/resource-manager/kubernetes/metadata.go:
> 
> import (
> ...
>     v1 "k8s.io/api/core/v1"
> ...
> )
> 
> const (
> ...
>     KeyNamespace = "cri-resource-manager.intel.com"
> ...
> )
> 
> func ResourceManagerKey(name string) string {
>     return KeyNamespace + "/" + name
> }
> 
> func InResourceManagerNamespace(key string) bool {
>     return strings.HasPrefix(key, KeyNamespace + "/")
> }
> 
> func IsResourceManagerLabel(key string) bool {
>     return InResourceManagerNamespace(key)
> }
> 
> func IsResourceManagerAnnotation(key string) bool {
>     return InResourceManagerNamespace(key)
> }
> 
> func IsResourceManagerResource(name v1.ResourceName) bool {
>     return InResourceManagerNamespace(string(name))
> }
> 
> ...
> ```

Makes sense. My thinking for originally keeping this functionality inside cache/container (it should've been under `cache/cache.go`, but anyway :smile:) was to keep it private until some other uses  would appear (and then refactor it out). I changed this now, adding `pkg/cri/resource-manager/kubernetes/kubernetes.go`. I only added the functionality that is needed as I didn't want to add any dead code.

I used **Resmgr** instead of **ResourceManager** in the code as it is shorter, needed less renaming and resmgr is already used in the name of the executable, for example. But, I don't really have strong feelings about the naming :wink: